packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -55,6 +57,24 @@ export function generateIsomorphicTests(
                         Enzyme.render(element);
                     });
                 }
+                if (classNameSkipList.includes(componentName)) {
```

Comment:
🤔 it's not so much about skipping as about elements where `className` is not on the root -- but they should still support `className`. 

maybe we can always have a unit test but if the component is in the list then we use `Enzyme.render(element).find(".test")`?

Reply:
Retitled to classNameChildList and made it so for anything on that list the test will search the whole component tree for the class.